config/routes.rb

```diff
@@ -292,31 +289,31 @@
     resources :textcompletes, only: [:index]
   end
 
-  root :to => 'pages#home'
+  root 'pages#home'
 
-  post '/products' => 'products#create', as: :products
+  post '/products', to: 'products#create', as: :products
 
   get '/products/:id', to: redirect(ProductRedirector.new), as: :full_product
   get '/products/:product_id/discussions/:id', to: redirect(ProductRedirector.new(:discussion)), as: :full_product_discussion
   get '/products/:product_id/tasks/:id', to: redirect(ProductRedirector.new(:task)), as: :full_product_task
 
-  get '/activities/:id' => 'activity#show'
-  get '/hotbounties' => 'hot_bounties#show'
+  get '/activities/:id', to: 'activity#show'
+  get '/hotbounties', to: 'hot_bounties#show'
 
-  get '/interests/:interest' => 'global_interests#toggle', as: :global_interests
-  get '/hello/:id' => 'hellos#show', as: :hello_user
+  get '/interests/:interest', to: 'global_interests#toggle', as: :global_interests
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.